coss/users/forms.py

```diff
@@ -29,11 +54,31 @@ def clean(self, *args, **kwargs):
             self._errors['site'] = self.error_class([error_msg])
         return cdata
 
+    def save(self, commit=True):
+        """Override save method of the form.
+
+        Fetch data from mozillians.org to add the avatar url in the profile.
+        """
+        obj = super(CossUserEditForm, self).save(commit=False)
+        obj = fetch_mozillians_data(obj)
```

Comment:
This is kind of misleading since it doesn't ``fetch_mozillians_data`` but adds values to an instance. I would either change the functionality or change the function name to ``populate_mozillians_data``.